runtests.sh

```
@@ -311,11 +309,13 @@ do
         --precommit)
             doPrecommit=true
         ;;
-        --pytype)
-            doPytypeFormat=true
+        --pyrefly)
+            doPyreflyFormat=true
         ;;
         --mypy)
             doMypyFormat=true
+        ;;
+            doMypyFormat=true
```

Shellcheck (0.11.0)

[error] Couldn't parse this case item. Fix to allow more checks.

(SC1073)